tracer/build/_build/Build.GitHub.cs

```diff
@@ -346,7 +346,13 @@ partial class Build
         {
             const string fixes = "Fixes";
             const string buildAndTest = "Build / Test";
-            const string changes = "Changes";
+            const string misc = "Miscellaneous";
```

Comment:
I wonder if this should be "Shared"?

Reply:
Don't know this one is trickier I'd say. But what I can do is come back to this when I have a look at the automatic labelling